gallery/src/components/NxNexusPageHeader/NxNexusPageHeaderPage.tsx

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxNexusPageHeaderExample from './NxNexusPageHeaderExample';
+import NxNexusPageHeaderCustomLogoExample from './NxNexusPageHeaderCustomLogoExample';
+
+const nxNexusPageHeaderExampleCode = require('!!raw-loader!./NxNexusPageHeaderExample').default;
+const nxNexusPageHeaderCustomLogoExampleCode = require('!!raw-loader!./NxNexusPageHeaderCustomLogoExample').default;
+
+const NxNexusPageHeaderPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        <code className="nx-code">NxNexusPageHeader</code> is a React component encapsulating the Sonatype Nexus
+        branded page header logo and structure. This header should only be used for products in the Sonatype Nexus
+        family.
+      </p>
+      <h3 className="nx-h3 nx-tile__subsection-header">Props</h3>
+      <table className="nx-table nx-table--gallery-props">
+        <thead>
+          <tr className="nx-table-row">
+            <th className="nx-cell nx-cell--header">Prop</th>
+            <th className="nx-cell nx-cell--header">Type</th>
+            <th className="nx-cell nx-cell--header">Required</th>
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="nx-table-row">
+            <td className="nx-cell">productInfo</td>
+            <td className="nx-cell">object</td>
+            <td className="nx-cell">No</td>
```

Comment:
Wait, why isn't this object required? I was thinking it was.  In what situation would you not have the product name at least?

Reply:
> Wait, why isn't this object required? I was thinking it was. In what situation would you not have the product name at least?

You're right and in the props in isn't optional either, updated in the docs: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/0654c36a0bdb2709bb8f4daa180cad0b5520eb79

Reply:
> You're right and in the props it isn't optional either, updated in the docs: 0654c36

Not sure how you figure that, it is optional in the type declarations in `types.ts` which is something else that needs to be adjusted.

Reply:
> > You're right and in the props it isn't optional either, updated in the docs: [0654c36](https://github.com/sonatype/sonatype-react-shared-components/commit/0654c36a0bdb2709bb8f4daa180cad0b5520eb79)
> 
> Not sure how you figure that, it is optional in the type declarations in `types.ts` which is something else that needs to be adjusted.

Right, fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/090ecaa4aa4c4877e2d822b161a7c0d68976b238